extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicOutputFormat.java

```diff
@@ -118,7 +118,8 @@ private void addBatch(RowData row) {
         StringJoiner value = new StringJoiner(this.fieldDelimiter);
         GenericRowData rowData = (GenericRowData) row;
         for(int i = 0; i < row.getArity(); ++i) {
-            value.add(rowData.getField(i).toString());
+            String field = rowData.getField(i) == null ? "" : rowData.getField(i).toString();
+            value.add(field);
```

Comment:
Do need to distinguish `null` and `empty string`?

Reply:
changed